src/js_collection.hpp

```diff
@@ -23,6 +23,9 @@
 #include "js_observable.hpp"
 
 #include "collection_notifications.hpp"
+#if REALM_ENABLE_SYNC
```

Comment:
Is there a case where the js binding has this set to false?

Reply:
I believe that we test both with and without sync on CI.

Reply:
Seems like wasted cycles if we don't release it after testing 😛

Reply:
Isn't realm-js on Linux built without sync unless you're installing one of the paid editions?

Reply:
I don't think so - we have a single package and feature tokens to unlock functionality.